feat: buttons,cards

ddivyansh18 opened the pull request:
Added buttons and cards to component library

ddivyansh18 committed:
feat: buttons,cards

Added buttons and cards to component library

anuj-kumary reviewed (changes requested):
Some suggestions on your card component.
- like button style is not that attractive give some padding 
- Increase a little bit width of every card so it looks clean
- In horizontal card add an icon below your content so it looks NYC

ddivyansh18 committed:
fix: card-button styles, horizontal card layout

- Moved the action icons in horizontal card layout at the bottom of text
- Removed pillbox styles from card buttons and vertically centered the text

ddivyansh18 commented:
@anuj-kumary I have made the requested changes, please approve or give additional feedbacks.

ddivyansh18 committed:
fix: card button styles, card shadow
